packages/eds-core-react/src/components/Popover/Popover.stories.tsx

```diff
@@ -255,3 +276,57 @@ Compact.parameters = {
     },
   },
 }
+
+export const WithCloseButton: Story<PopoverProps> = () => {
+  const [isOpen, setIsOpen] = useState<boolean>(false)
+  const anchorRef = useRef<HTMLButtonElement>(null)
+
+  const openPopover = () => setIsOpen(true)
+  const closePopover = () => setIsOpen(false)
+
+  return (
+    <EdsProvider>
+      <StoryCenter>
+        <Button
+          id="click-popover-anchor"
+          aria-controls="click-popover"
+          ref={anchorRef}
+          onClick={openPopover}
+        >
+          Click to activate
+        </Button>
+
+        <Popover
+          id="click-popover"
+          aria-expanded={isOpen}
+          anchorEl={anchorRef.current}
+          onClose={closePopover}
+          open={isOpen}
+        >
+          <Popover.Header>
+            <Popover.Title>Title</Popover.Title>
+            <Button
+              style={{ height: '32px', width: '32px' }}
+              variant="ghost_icon"
+              title="Close popover"
+              onClick={closePopover}
+            >
+              <Icon name="close" data={close} size={24} />
+            </Button>
+          </Popover.Header>
+          <Popover.Content>
+            <Typography variant="body_short">Content</Typography>
+          </Popover.Content>
+        </Popover>
+      </StoryCenter>
+    </EdsProvider>
+  )
+}
+
+WithCloseButton.parameters = {
+  docs: {
+    description: {
+      story: 'Description comes here ',
```

Comment:
I suppose this needs to be filled out still, but I am not sure myself in what scenarios it is recommended to use closebutton. @mimarz ? (Maybe as a security measure in a dense page full of interactive stuff you risk clicking on accidentally with clickoutside)

Reply:
Looking at the documentation for [Popover](https://eds.equinor.com/0b0c666ab/p/8809df-popover/b/49f4b2), its meant for interactive use-cases.

Reply:
Oddbjørn's argument seems reasonable , but still user can use esc button to avoid collision with other clickable elements?
I still wander if the close button is needed, especially when there are two way to close it? I would see it as a informative for the user. 🤔

Reply:
What if we say: Popover with close button can be use in a dense page, a page filled with many clickable elements to avoid collision with them. Close button can be useful when Popover do not contain any clickable elements.
Something along those lines?

Reply:
Maybe we can avoid the reasoning for now and say something generic like, 'In some cases an explicit close button is recommended. Here is an example of how this can be implemented in the Header'

Reply:
Nice @oddvernes